hydrolib/core/basemodel.py

```diff
@@ -132,8 +159,15 @@ def is_file_link(self) -> bool:
     def validate(cls: Type["FileModel"], value: Any):
         # Enable initialization with a Path.
         if isinstance(value, (Path, str)):
+            filepath = Path(value)
+
+            if not filepath.is_absolute():
+                folder = context_dir.get(None)
```

Comment:
This folder could be None right?

Reply:
Might be good to keep the logic for the context_dir in one spot, maybe move the set part to here as well?